ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-path-selection.j2

```diff
@@ -0,0 +1,107 @@
+{#
+ Copyright (c) 2023 Arista Networks, Inc.
+ Use of this source code is governed by the Apache License 2.0
+ that can be found in the LICENSE file.
+#}
+{# eos - router path-selection #}
+{% if router_path_selection is arista.avd.defined %}
+!
+router path-selection
+{%     if router_path_selection.peer_dynamic_source is arista.avd.defined %}
+   peer dynamic source {{ router_path_selection.peer_dynamic_source }}
+{%     endif %}
+{#  path group #}
+{%     for path_group in router_path_selection.path_groups | arista.avd.natural_sort('name') %}
+{%         set path_group_def = "path-group " ~ path_group.name %}
+{%         if path_group.id is arista.avd.defined %}
+{%             set path_group_def = path_group_def ~ " id " ~ path_group.id %}
+{%         endif %}
+   !
+   {{ path_group_def }}
+{%         if path_group.ipsec_profile is arista.avd.defined %}
+      ipsec profile {{ path_group.ipsec_profile }}
+{%         endif %}
+{%         if path_group.flow_assignment is arista.avd.defined %}
+      flow assignment {{ path_group.flow_assignment }}
+{%         endif %}
+{#      local interfaces #}
+{%         for local_interface in path_group.local_interfaces | arista.avd.natural_sort('name') %}
+{%             set local_interface_def = "local interface " ~ local_interface.name %}
+{%             if local_interface.public_address is arista.avd.defined %}
+{%                 set local_interface_def = local_interface_def ~ " public address " ~ local_interface.public_address %}
+{%             endif %}
+      !
+      {{ local_interface_def }}
+{%             if local_interface.stun is arista.avd.defined %}
+         stun server-profile {{ local_interface.stun.server_profiles | join(" ") }}
+{%             endif %}
+{%         endfor %}
+{#      local ips #}
+{%         for local_ip in path_group.local_ips | arista.avd.natural_sort('ip_address') %}
+{%             set local_ip_def = "local ip " ~ local_ip.ip_address %}
+{%             if local_ip.public_address is arista.avd.defined %}
+{%                 set local_ip_def = local_ip_def ~ " public address " ~ local_ip.public_address %}
+{%             endif %}
+      !
+      {{ local_ip_def }}
+{%             if local_ip.stun is arista.avd.defined %}
+         stun server-profile {{ local_ip.stun.server_profiles | join(" ") }}
+{%             endif %}
+{%         endfor %}
+{#      peer dynamic #}
+{%         if path_group.dynamic_peers is arista.avd.defined %}
+      !
+      peer dynamic
+{%             if path_group.dynamic_peers.ip_local is arista.avd.defined(true) %}
+         ip local
+{%             endif %}
+{%             if path_group.dynamic_peers.ipsec_disabled is arista.avd.defined(true) %}
```

Comment:
this should be updated to this:
- if set to true set `ipsec`
- if set to false set `ipsec disable`

Reply:
[a8bcad1](https://github.com/aristanetworks/ansible-avd/pull/3203/commits/a8bcad1df5579a690fc2a3dc5780901f6e9f8763)